nautobot/extras/api/customfields.py

```diff
@@ -52,18 +52,10 @@ def _get_custom_fields(self):
         return self._custom_fields
 
     def to_representation(self, obj):
-        # 2.0 TODO: #824 use cf.slug as lookup key instead of cf.name
-        return {cf.slug: obj.get(cf.name) for cf in self._get_custom_fields()}
+        return {cf.key: obj.get(cf.key) for cf in self._get_custom_fields()}
```

Comment:
Does this need to be?
```suggestion
        return {cf.key: obj.get(cf.label) for cf in self._get_custom_fields()}
```

Reply:
Nah actually the `obj.cf` has `cf.key` as its key as well